Assets/MixedRealityToolkit/InputSystem/Sources/BaseGenericInputSource.cs

```diff
@@ -8,6 +8,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.XR.WSA.Input;
```

Comment:
Why?

Reply:
Temp only. Need to separate device state to Mrtk state (so) but was to lazy to do it atm 😎